forge-gui-desktop/src/main/java/forge/screens/match/controllers/CCombat.java

```diff
@@ -78,13 +78,19 @@ private static String getCombatDescription(final CombatView localCombat, final G
         }
 
         display.append("\n");
-
+        PlayerView controller = null;
         if (defender instanceof CardView) {
-            final PlayerView controller = ((CardView) defender).getController();
-            display.append(Lang.getInstance().getPossesive(controller.getName())).append(" ");
+            controller = ((CardView) defender).getController();
+            if (controller == null)
```

Comment:
I don't like such null checks that just hide the bug instead of really fixing the cause 🤔

Reply:
So I just leave it crashing why it didn't have controller? I can just revert it if you want.

Reply:
Nah that's not what I meant
since in this case the View is supposed to represent the ingame object it shouldn't be possible to have no controller during combat, so that's what needs debugging

Reply:
play a network game and most object will have null, I mean It's a hard to reproduce on single player but It shouldn't crash the GUI or you can't finish any game. If its really null then it will crash somewhere else.